doc/libpmemfile.1.md

```diff
@@ -1,9 +1,9 @@
 ---
 layout: manual
 Content-Style: 'text/css'
-title: libpmemfile(1)
-header: PMEMFILE
-date: pmemfile API version 0.1.0
+title: pmemfile(1)
+header: PMEMFile
+date: pmemfile API version 1.0.0
```

Comment:
So we would already call it 1.0 ?
I'm not quite confident about that.
How about waiting till most of the interfaces described in this manual work already  - e.g. pmemfile_truncate functioning?